compiler/infra/OMRCfg.hpp

```diff
@@ -60,6 +60,10 @@ namespace TR { class ResolvedMethodSymbol; }
 namespace TR { class TreeTop; }
 template <class T> class TR_Array;
 
+// Needed for friend declaration
+// OMR::Block needs access to TR::Region _internalRegion in OMR::CFG
+namespace OMR { class Block; }
```

Comment:
Out of curiosity, is there a reason for not using a public getter for `_internalRegion` instead of making `OMR::Block` a friend (assuming that's the only reason it's a friend)?

Reply:
I think it would be best not to make `_internalRegion` accessible through a public getter because that would allow the people to store things not related to CFG in `_internalRegion`.
But if you think it is a better idea than making it a friend, sure.

Reply:
I think either design would work so we can go with an API to get the region if that is cleaner, but make sure the API is get only with no setter so the allocation region stays consistent.

Reply:
So, just to be clear, I should change the API from a friend to a getter, right?

Reply:
Yes that is the conclusion of the review from @Leonardo2718 and I am fine with that